src/app/buttons/button-loading.html

```diff
@@ -6,23 +6,23 @@
   -->
 
 <h4>Loading Buttons</h4>
-<button [clrLoading]="validateLoading" class="btn btn-info-outline" (click)="validateDemo()">Validate</button>
-<button [clrLoading]="submitLoading" type="submit" class="btn btn-success-outline" (click)="submitDemo()">Submit</button>
+<button [clrLoading]="validateState" class="btn btn-primary" (click)="validateDemo()">Validate</button>
+<button [clrLoading]="submitState" type="submit" class="btn btn-success-outline" (click)="submitDemo()">Submit</button>
 
 <pre>
```

Comment:
Can we remove the code snippets from the demos? We have decided to do that across all demos now. We will also get rid of prismjs from clarity & our dev app.

Reply:
Yup will remove!